js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,12 @@
+const animalSort = animals => {
+  return animals.sort((firstAnimal, secondAnimal) => {
+    if (firstAnimal.numberOfLegs === secondAnimal.numberOfLegs) {
+      if (firstAnimal.name > secondAnimal.name) return 1;
+      else if (firstAnimal.name < secondAnimal.name) return -1;
+      else return 0;
```

Comment:
Although not compulsory, braces in single line conditionals are recommended. Not having them can lead to severe issues which can go unseen if the codebase is expanding.
Refer to, https://stackoverflow.com/questions/4797286/are-braces-necessary-in-one-line-statements-in-javascript

Reply:
Updated with changes.